src/pages/sample-page.js

```diff
@@ -9,6 +9,12 @@ import { Link } from 'gatsby';
 // UI lib components
 import { Container } from 'react-grid-system';
 
+/* ------------------------------ Static Image ------------------------------ */
```

Comment:
@raed-emiketic You should install Comment Divider (which is identified as a recommended extension us under .vscode > .extensions, and visible on the sidebar in VSCode): https://marketplace.visualstudio.com/items?itemName=stackbreak.comment-divider
This way we all have the same comment format

If this already the case, that's fine

Reply:
already installed and worked with